spark/src/main/scala/io/substrait/spark/expression/FunctionMappings.scala

```diff
@@ -47,6 +47,7 @@ class FunctionMappings {
     s[GreaterThan]("gt"),
     s[GreaterThanOrEqual]("gte"),
     s[EqualTo]("equal"),
+    s[EqualNullSafe]("is_not_distinct_from"),
```

Comment:
EqualNullSafe matches the behaviour of `is_not_distinct_from`

https://github.com/apache/spark/blob/31a411773a3e97adb833289f8c695b37802cfedb/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/predicates.scala#L1070-L1094